mensor/measures/types.py

```diff
@@ -380,7 +380,9 @@ def __init__(self, name, unit_type=None, via=None, external=False, private=False
             if self.ALLOW_ALL_ATTRIBUTES or attr in self.EXTRA_ATTRIBUTES:
                 setattr(self, attr, value)
             else:
-                raise KeyError("No such attribute {}.".format(attr))
+                raise AttributeError(
+                    "Cannot initialize {}<{}> with attribute {}.".format(self.__class__.__name__, self.name, attr)
```

Comment:
Can you quote the attribute name in single quotes?

Reply:
done